content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -254,3 +254,8 @@ Example: `node.cloudprovider.kubernetes.io/shutdown:NoSchedule`
 
 If a Node is in a cloud provider specified shutdown state, the Node gets tainted accordingly with `node.cloudprovider.kubernetes.io/shutdown` and the taint effect of `NoSchedule`.
 
+## kubectl.kubernetes.io/default-container
+
+Example: `kubectl.kubernetes.io/default-container:container1`
```

Comment:
I'd pick a more plausible name and format it like other example annotations. For example:
```suggestion
Example: `kubectl.kubernetes.io/default-container: "front-end-app"`
```